server/events/command_runner.go

```diff
@@ -354,6 +360,24 @@ func (c *DefaultCommandRunner) RunCommentCommand(baseRepo models.Repo, maybeHead
 		return
 	}
 
+	// Update the combined plan or apply commit status to pending
+	switch cmd.Name {
```

Comment:
Not sure why this is a switch statement, are we intentionally not doing command.Plan? The commented-out code makes it a bit hard to understand the intent here